MAINTAINERS

```diff
@@ -20,7 +20,10 @@ Jonas Östanbäck <jonas.ostanback@gmail.com> (@cez81)
 David Schneiderbauer <dschneiderbauer@gmail.com> (@daviian)
 Peter Žeby <morlinest@gmail.com> (@morlinest)
 Matti Ranta <matti@mdranta.net> (@techknowlogick)
-Michael Lustfield <mtecknology@debian.org> (@MTecknology)
```

Comment:
Why deleting this line ? 
I think you simply put the same MAINTAINERS file across all PRs but this one seems to have one more maintainer.

Reply:
MTecknology sent a PR to remove his name from the main repo, but not the others. I've asked Jonas to include all changes from main repo in his PRs to the secondary repos.

Reply:
I since see https://github.com/go-gitea/blog/pull/83#discussion_r244766345 and https://github.com/go-gitea/gitea/pull/5599